website/docs/r/domain.html.markdown

```diff
@@ -25,6 +25,8 @@ The following arguments are supported:
 
 * `name` - (Required) A unique name for the resource, required by libvirt.
   Changing this forces a new resource to be created.
+* `description` - (Optional) The description for domain.
```

Comment:
last nitpick. From User experience point, I would like that we state why this could be useful.

E.g why one should add the description?

We should state also `(Optional) The description for domain. Useful in case of XX for XX bla`

Reply:
To be honest... The description field does not have any technical purpose, it's just notes, so it's very hard for me to find use cases...

Reply:
why do you want to add it then? :thinking:

Reply:
I use this field for dynamic inventory for ansible. But this is my custom script and no one else use it... The libvirt documentation https://libvirt.org/formatdomain.html#elementsMetadata says that:
```
 description
    The content of the description element provides a human readable description of the virtual machine.
    This data is not used by libvirt in any way, it can contain any information the user wants. 
```

Reply:
ok . I think we can paste and copy libvirt definition

```    The content of the description element provides a human readable description of the virtual machine.
    This data is not used by libvirt in any way, it can contain any information the user wants. 
```

Reply:
done